rules/netskope_rules/netskope_many_deletes.yml

```diff
@@ -0,0 +1,67 @@
+AnalysisType: rule
+RuleID: "Netskope.ManyDeletes"
+DisplayName: "Netskope Many Objects Deleted"
+AlertTitle: "[{user}] deleted many objects in a short time"
+Detection:
+  - All:
+    - KeyPath: audit_log_event
+      Condition: Contains
+      Value: Delete
```

Comment:
Does the `Value` need to be wrapped in quotes or is that only for multi-word strings with spaces?

Reply:
As long as it's recognized as a string in the yaml syntax it should work https://docs.panther.com/detections/rules/yaml/match-expression#scalar-matching-conditions